paying_for_college/models.py

```diff
@@ -243,6 +266,41 @@ def primary_alias(self):
             return 'Not Available'
 
 
+class Notification(models.Model):
+    """record of a disclosure verification"""
+    institution = models.ForeignKey(School)
+    oid = models.CharField(max_length=40)
+    timestamp = models.DateTimeField()
+    errors = models.CharField(max_length=255)
+    email = models.CharField(max_length=255, blank=True)
+
+    def __unicode__(self):
+        return "{0} {1}".format(self.oid, self.institution.primary_alias)
+
+    def notify_school(self):
+        payload = {
+            'oid': self.oid,
+            'time': self.timestamp.isoformat(),
+            'errors': self.errors
+        }
+        school = self.institution
+        # we prefer to use endpount notification, so use it first if existing
+        if school.contact:
+            if school.contact.endpoint:
+                requests.post(school.contact.endpoint, data=payload)
+                return "school notified via endpoint"
+            elif school.contact.contact:
+                send_mail("CFPB disclosure notification",
```

Comment:
Send mail is awesome but I don't see any configs in settings.py. Is there a "college-costs" version of send_mail method? Are you going to send out more emails for any reason? If so, I would consider refactoring the send_mail into it's own module and use a custom send_mail method that gives you a custom handling. For example, you can create your custom mail_method to just accept a template name and a context and it can have a centralized template selection (or even django based templates). All emails will automatically come from no-reply@cfpb.gov etc.